src/quacc/runners/ase.py

```diff
@@ -362,3 +418,80 @@ def _set_sella_kwargs(self, optimizer_kwargs: dict[str, Any]) -> None:
 
         if not self.atoms.pbc.any() and "internal" not in optimizer_kwargs:
             optimizer_kwargs["internal"] = True
+
+    @staticmethod
+    def _fix_deprecated_md_params(dynamics_kwargs: dict[str, Any]) -> None:
+        """
+        Helper function to handle deprecated MD parameters.
+
+        Parameters
+        ----------
+        dynamics_kwargs
+            Dictionary of keyword arguments for the molecular dynamics calculation.
+
+        Returns
+        -------
+        None
+        """
+
+        if "trajectory" in dynamics_kwargs:
+            msg = "Quacc does not support setting the `trajectory` kwarg."
+            raise ValueError(msg)
+
+        if "temperature" in dynamics_kwargs or "temp" in dynamics_kwargs:
+            LOGGER.warning(
+                "In quacc `temperature`, `temp` and `temperature_K` are"
+                " equivalent and will be interpreted in Kelvin."
+            )
+            dynamics_kwargs["temperature_K"] = dynamics_kwargs.pop(
+                "temperature", None
+            ) or dynamics_kwargs.pop("temp", None)
+
+        if "pressure" in dynamics_kwargs:
+            LOGGER.warning(
+                "In quacc `pressure` and `pressure_au` are equivalent and will be"
+                " interpreted in GPA."
+            )
+            dynamics_kwargs["pressure_au"] = dynamics_kwargs.pop("pressure")
+
```

Comment:
Do we need all these checks, or can we just rely on the fact that the user should be using the up-to-date names and if they don't, ASE will warn them or a crash will happen.

Reply:
The thing is that, parameters are a very **big** mess currently, for example for NVT Berendsen we have:

```
temperature: float
The desired temperature, in Kelvin.

temperature_K: float
Alias for temperature
```
And for :

```
temperature: float (deprecated)
The desired temperature in eV.

temperature_K: float
The desired temperature in K.
```

Sometimes `temperature` is in eV, sometimes in K, you don't know, surprise!

I know it's a lot of time to ask from you but your support on https://gitlab.com/ase/ase/-/merge_requests/3391 (probably not everything in this PR is good to take) would be more than welcome.

If/when this goes, I would rewrite the entire MD doc and we can finally move on with X-years old deprecated keywords and have user-friendliness and consistency.

Reply:
Ooof, yeah that is a mess! Thanks for the reminder.

Let's see how the discussion plays out and then I am happy to proceed how you suggest. I am happy to review on ASE's side if/when we get a blessing from Ask on certain components.

Reply:
Going to postpone changes here for the time being.